server.go

```diff
@@ -1614,11 +1611,8 @@ func newServer(cfg *Config, listenAddrs []net.Addr,
 			}
 
 			// If not we scale according to channel size.
-			delay := uint16(btcutil.Amount(maxRemoteDelay) *
-				chanAmt / MaxFundingAmount)
-			if delay < minRemoteDelay {
-				delay = minRemoteDelay
-			}
+			delay := max(uint16(btcutil.Amount(maxRemoteDelay)*
```

Comment:
incorrect formatting.

Reply:
I re-read the section on "Wrapping long function calls" in development_guidelines.md and realized the issue. 

Have made the necessary modifications. And I am happy to submit code for such a rigorously defined project (though I should familiarize myself with the guidelines beforehand).

Reply:
So you checked the "The change obeys the [Code Documentation and Commenting](https://github.com/lightningnetwork/lnd/blob/master/docs/development_guidelines.md#code-documentation-and-commenting) guidelines, and lines wrap at 80." checkbox without actually reading them first?
Gotcha, thanks for wasting my time. Not going to review this PR any further.

Reply:
Clearly, I have read it; otherwise, I wouldn't have made the corresponding changes. While I misunderstood, I have submitted code to many Go projects and haven't encountered such strict guidelines, but I am eager to learn. 

Apologize for making you feel like your time was wasted and  the confusion caused by the misunderstandings.